src/commands/blockchain/transactionsByHeight.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ *
+ * Copyright 2018 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import {command, ExpectedError, metadata, option} from 'clime';
+import {BlockHttp, Order, QueryParams} from 'nem2-sdk';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+import {TransactionService} from '../../service/transaction.service';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'h',
+        description: 'Block height',
+    })
+    height: number;
+
+    @option({
+        flag: 's',
+        description: 'Page size between 10 and 100, defaults to 10',
+    })
+    pageSize: number;
+
+    @option({
+        flag: 'i',
+        description: 'Id after which we want objects to be returned, defaults to empty',
+    })
+    id: string | undefined;
```

Comment:
Why do you add ``|undefined``?

Reply:
Because I saw a statement about `undefined` in [nem2-sdk-typescript-javascript](https://github.com/nemtech/nem2-sdk-typescript-javascript/blob/master/src/infrastructure/api/blockRoutesApi.ts#L227), if I don't need it, I will remove the undefined type later.

Reply:
It means that the filed is optional, I would opt to remove the "undefined" from the input option and add it if necessary at line 80.